### PR TITLE
Keep the same format with captializing

### PR DESCRIPTION
**Pull request by yuexiao-wang**
Keep the same format with captializing

Signed-off-by: yuexiao-wang wang.yuexiao@zte.com.cn

### CODE REVIEW

**Comment by SvenDowideit** [comment]
i don't know - capitalization implies each point is a sentence - which also means it'd need a full stop.

I used to lean more to bullet points being a way to represent parts of a sentence, and would use comma's

I guess this is a style question - @sfsmithcha ?

File: docs/swarm/join-nodes.md [hunk 1]
```diff
@@ -109,5 +109,5 @@ This node joined a swarm as a manager.
 
 ## Learn More
 
-* `swarm join`[command line reference](../reference/commandline/swarm_join.md)
+* `Swarm join`[command line reference](../reference/commandline/swarm_join.md)
```
Comment (sfsmithcha) [inline docs/swarm/join-nodes.md]: this is incorrect. command is `swarm join`

File: docs/swarm/swarm-tutorial/index.md [hunk 2]
```diff
@@ -33,20 +33,20 @@ If you’re brand new to Docker, see [About Docker Engine](../../index.md).
...
 The tutorial uses three networked host machines as nodes in the swarm. These can
 be virtual machines on your PC, in a data center, or on a cloud service
 provider. This tutorial uses the following machine names:
 
-* manager1
-* worker1
-* worker2
+* Manager1
+* Worker1
+* Worker2
```
Comment (sfsmithcha) [inline docs/swarm/swarm-tutorial/index.md]: These are machine names for the tutorial and are always used lowercase.

**Comment by sfsmithcha** [comment]
Thank you for your contribution @yuexiao-wang . I'm closing this for a couple of reasons:

Commands and machine names should reflect the command line and also the machine names.
Capitalization is not required in the case of these bullets - they are continuations of a sentence, not the beginning of a new sentence.

**Pull request closed without merging**